client/my-sites/plan-features/index.jsx

```diff
@@ -226,11 +226,11 @@ class PlanFeatures extends Component {
 						current={ current }
 						currentPlanTitle={ sitePlan.product_name_short }
 						discountPrice={ discountPrice }
+						isJetpackSite={ site.jetpack }
```

Comment:
In the child component, this was `!! site.jetpack`. Should always be a bool, though, right?

Reply:
I think so. It certainly won't cause any trouble for the actual logic, but might break `propTypes` checks.

I might leave it until we see start seeing `propType` errors. `undefined` shouldn't be a problem, since the prop type isn't `isRequired`. `null` could be a problem tho. I just hope we never fall back to `null`...

Reply:
It comes from the API directly so should be fine.